app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,7 @@
 
   </head>
 
-  <body>
+  <body class="<%= controller_name %> <%= action_name %>">
```

Comment:
https://github.com/lucasmazza/page.js
you can use this guy to initialize your componentes for each page, I used it before and it was Pretty good.